pkgs/applications/blockchains/wownero.nix

```diff
@@ -1,40 +1,26 @@
 { lib, stdenv, fetchgit, cmake, boost, miniupnpc_2, openssl, unbound
-, readline, libsodium, rapidjson, fetchurl
+, readline, libsodium, rapidjson, fetchurl, zeromq
 }:
 
 with lib;
 
-let
-  randomwowVersion = "1.1.7";
-  randomwow = fetchurl {
-    url = "https://github.com/wownero/RandomWOW/archive/${randomwowVersion}.tar.gz";
-    sha256 = "1xp76zf01hnhnk6rjvqjav9n9pnvxzxlzqa5rc574d1c2qczfy3q";
-  };
-in
-
 stdenv.mkDerivation rec {
   pname = "wownero";
-  version = "0.8.0.1";
+  version = "0.9.2.2";
 
   src = fetchgit {
     url = "https://git.wownero.com/wownero/wownero.git";
     rev = "v${version}";
-    sha256 = "15443xv6q1nw4627ajk6k4ghhahvh82lb4gyb8nvq753p2v838g3";
-    fetchSubmodules = false;
+    sha256 = "055hiz8xjkh6v5cx5igwahkzk8cp8z014aw7hl5q8vbpmhv9jfjf";
+    fetchSubmodules = true;
```

Comment:
Please unvendor the old unbound and rapidjson.

Reply:
Hmm, you want me to update the project submodules or is it something that can be done within nix?

Reply:
OK, I see what you are saying. Nix's packages are fairly up to date, there's no need to link against the vendored unbound and rapidjson. The fetchSubmodues part is for another dependency RandomWow. It used to be manually fetched as you can see in the diff, but since the archive format changed while the project moved to a different hosting solution, the unpacked directory also changed, which wouldn't work with a simple swap of hash. Hope this makes it clear :D

Reply:
I know exactly what you are getting, I completely agree. That's why randomwow was manually fetched in the first place to make the build precise. Currently this is good enough for me though, do you want to improve the patch or I can drop it, it's not a hard push :D

Reply:
I know exactly what you are getting, I completely agree. That's why randomwow was manually fetched in the first place to make the build precise. Currently this is good enough for me though, do you want to improve the patch or I can drop it, it's not a hard push :D

Reply:
I don't have the time to improve it. Fetching the one dependency is the cleanest way here and keeping it like it was is a good idea.

Reply:
OK, hopefully someone will figure out what was changed in the archive format and submit a better PR.

Reply:
The download link for a gitea archive is https://git.wownero.com/wownero/RandomWOW/archive/1.1.8.tar.gz which wasn't to hard to figure out.

Reply:
Fix it yourself

Reply:
I do not have the time to do that.